source/uwp/Renderer/lib/XamlHelpers.h

```diff
@@ -28,8 +28,83 @@ namespace AdaptiveNamespace::XamlHelpers
         return solidColorBrushAsBrush;
     }
 
+    void WireButtonClickToAction(_In_ ABI::Windows::UI::Xaml::Controls::IButton* button,
```

Comment:
>WireButtonClickToAction [](start = 9, length = 23)

What's the scope of ActionHelpers? Should this go there?

Reply:
Also, potentially:
WrapInTouchTarget
HandleSelectAction
BuildActions
BuildActionSetHelper

---
In reply to: [325846545](https://github.com/microsoft/AdaptiveCards/pull/3457#discussion_r325846545) [](ancestors = 325846545)